packages/indexer/src/utils/__tests__/rpc.helpers.test.ts

```diff
@@ -0,0 +1,51 @@
+import { ApiPromise, WsProvider } from "@polkadot/api"
+import { keccakAsU8a } from "@polkadot/util-crypto"
+import { getEvmTokenDecimals, getSubstrateTokenDecimals } from "../rpc.helpers"
+
+describe("RPC Helpers", () => {
```

Comment:
These tests are not useful, add an integration test that checks that it indexes token prices in the sdk tests